ansible/roles/products/common/dolphin-icons/tasks/main.yml

```diff
@@ -92,6 +92,13 @@
   changed_when: false
   with_items: "{{ file_list.files }}"
 
+- name: Create shadow_hand_launcher folder
```

Comment:
What is this?

Reply:
but you are on dolphin role now? This should have been created many steps above. If not thats not the place to add it